omniscidb/QueryEngine/QueryExecutionContext.cpp

```diff
@@ -124,9 +124,9 @@ ResultSetPtr QueryExecutionContext::groupBufferToDeinterleavedResults(
   auto deinterleaved_result_set =
       std::make_shared<ResultSet>(result_set->getTargetInfos(),
                                   std::vector<ColumnLazyFetchInfo>{},
-                                  std::vector<std::vector<const int8_t*>>{},
-                                  std::vector<std::vector<int64_t>>{},
-                                  std::vector<int64_t>{},
+                                  std::make_shared<const ColBuffers>(),
```

Comment:
Why not simply `nullptr`? Column buffers are supposed to be used for lazy columns only.